channel_simulator/PathlossPair[] path_array
